From rdunlap@xenotime.net Sat Aug 26 10:55:29 2006
Date: Sat, 26 Aug 2006 10:58:24 -0700
From: "Randy.Dunlap" <rdunlap@xenotime.net>
Cc: gregkh <greg@kroah.com>, akpm <akpm@osdl.org>
Subject: adutux: fix printk format warnings
Message-Id: <20060826105824.935b5bd3.rdunlap@xenotime.net>

From: Randy Dunlap <rdunlap@xenotime.net>

Fix printk format warnings:
drivers/usb/misc/adutux.c:388: warning: format ‘%d’ expects type ‘int’, but argument 3 has type ‘size_t’
drivers/usb/misc/adutux.c:541: warning: format ‘%d’ expects type ‘int’, but argument 3 has type ‘size_t’
drivers/usb/misc/adutux.c:590: warning: format ‘%d’ expects type ‘int’, but argument 3 has type ‘size_t’
drivers/usb/misc/adutux.c:593: warning: format ‘%d’ expects type ‘int’, but argument 3 has type ‘size_t’
drivers/usb/misc/adutux.c:598: warning: format ‘%d’ expects type ‘int’, but argument 3 has type ‘size_t’
drivers/usb/misc/adutux.c:598: warning: format ‘%d’ expects type ‘int’, but argument 4 has type ‘size_t’
drivers/usb/misc/adutux.c:598: warning: format ‘%d’ expects type ‘int’, but argument 5 has type ‘size_t’

Signed-off-by: Randy Dunlap <rdunlap@xenotime.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/misc/adutux.c |   12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

--- gregkh-2.6.orig/drivers/usb/misc/adutux.c
+++ gregkh-2.6/drivers/usb/misc/adutux.c
@@ -201,7 +201,7 @@ static void adu_interrupt_in_callback(st
 				dev->interrupt_in_buffer, urb->actual_length);
 
 			dev->read_buffer_length += urb->actual_length;
-			dbg(1," %s reading  %d ", __FUNCTION__,
+			dbg(2," %s reading  %d ", __FUNCTION__,
 			    urb->actual_length);
 		} else {
 			dbg(1," %s : read_buffer overflow", __FUNCTION__);
@@ -385,7 +385,7 @@ static ssize_t adu_read(struct file *fil
 	unsigned long flags;
 	DECLARE_WAITQUEUE(wait, current);
 
-	dbg(2," %s : enter, count = %d, file=%p", __FUNCTION__, count, file);
+	dbg(2," %s : enter, count = %Zd, file=%p", __FUNCTION__, count, file);
 
 	dev = file->private_data;
 	dbg(2," %s : dev=%p", __FUNCTION__, dev);
@@ -538,7 +538,7 @@ static ssize_t adu_write(struct file *fi
 	int retval = 0;
 	int timeout = 0;
 
-	dbg(2," %s : enter, count = %d", __FUNCTION__, count);
+	dbg(2," %s : enter, count = %Zd", __FUNCTION__, count);
 
 	dev = file->private_data;
 
@@ -587,15 +587,15 @@ static ssize_t adu_write(struct file *fi
 			goto exit;
 		}
 
-		dbg(4," %s : in progress, count = %d", __FUNCTION__, count);
+		dbg(4," %s : in progress, count = %Zd", __FUNCTION__, count);
 
 		} else {
-			dbg(4," %s : sending, count = %d", __FUNCTION__, count);
+			dbg(4," %s : sending, count = %Zd", __FUNCTION__, count);
 
 			/* write the data into interrupt_out_buffer from userspace */
 			buffer_size = le16_to_cpu(dev->interrupt_out_endpoint->wMaxPacketSize);
 			bytes_to_write = count > buffer_size ? buffer_size : count;
-			dbg(4," %s : buffer_size = %d, count = %d, bytes_to_write = %d",
+			dbg(4," %s : buffer_size = %Zd, count = %Zd, bytes_to_write = %Zd",
 			    __FUNCTION__, buffer_size, count, bytes_to_write);
 
 			if (copy_from_user(dev->interrupt_out_buffer, buffer, bytes_to_write) != 0) {
